website/content/docs/deprecation/index.mdx

```diff
@@ -10,6 +10,45 @@ description: |-
 
 This announcement page is maintained and updated periodically to communicate important decisions made concerning End of Support(EoS) for Vault features as well as features we have removed or disabled from the product. We document the removal of features, enable the community with a plan and timeline for eventual deprecations, and supply alternative paths to explore and evaluate to minimize business disruptions. If you have questions or concerns about a deprecated feature, please create a topic on [the community forum](https://discuss.hashicorp.com/c/vault/30) or raise a ticket with your support team. Please refer to the [FAQ](/docs/deprecation/faq) page for frequently asked questions concerning Vault feature deprecations.
 
+## Deprecation Status
+
+As of version 1.12, Vault implements a multi-phased approach to deprecation. The intent of this approach is to provide sufficient warning that a feature will be removed and safe handling of stored data when the associated feature has been removed.
+
+The phases of deprecation are also known as "Deprecation Status". These statuses are currently reflected in builtin plugins and are exposed via the Vault `auth`, `secrets`, and `plugins` CLI/API endpoints. For more information, refer to the corresponding documentation.
+
+The four phases of deprecation are: `Supported`, `Deprecated`, `Pending Removal`, and `Removed`.
+
+### Supported
+
+This is the default status and reflects a feature which is still supported. There is no unique behavior or functionality associated with this status.
+
+### Deprecated
+
+This status reflects a feature which has been marked for deprecation in a later release of Vault. This is the first phase of the deprecation process. A status of `Deprecated` has two effects:
+
+1) All attempts to utilize this feature will result in `Warn`-level messages in the Vault log.
+2) All attempts to access an endpoint which supports this feature will result in `warnings` in the Response.
+
+### Pending Removal
+
+This status reflects a feature which has been officially deprecated in this release of Vault. This is the first phase in the process that fundamentally alters the behavior of Vault. The effects are two-fold:
+
+1) After an upgrade, any existing `Pending Removal` mount will cause Vault to log an `Error`-level message in the Vault log and automatically shutdown.
```

Comment:
Up until this point, all the text has been speaking about deprecation status of *features* and and *endpoints*.  Now suddenly we're speaking about it solely in terms of mounts.  Maybe we should keep the focus on mounts throughout?

Reply:
I'd hazard a guess that focusing on features would be more user friendly than something like "mounts" which is kind of second-level terminology.

Reply:
Thank you both. I can make the wording more consistent around "feature" as opposed to "mount".